### PR TITLE
Update services routes

### PR DESCRIPTION
**Pull request by fao89**
https://github.com/openstack-k8s-operators/openstack-operator/pull/457

### CODE REVIEW

File: docs/openstack/glance_adoption.md [hunk 1]
```diff
@@ -88,11 +94,6 @@ spec:
       storageClass: "local-storage"
       storageRequest: 10G
       glanceAPIInternal:
-        externalEndpoints:
-        - endpoint: internal
-          ipAddressPool: internalapi
-          loadBalancerIPs:
-          - 172.17.0.80
```
Comment (stuggi) [inline docs/openstack/glance_adoption.md:95]: to match the old example you may also want to have the override for ceph?

Comment (fao89) [reply docs/openstack/glance_adoption.md:95]: could you please give me an example?
I didn't find what is the proper way to override ceph

**Comment by fao89** [comment]
recheck

File: docs/openstack/glance_adoption.md (hunk 1 above)
Comment (stuggi) [reply docs/openstack/glance_adoption.md:95]: sorry, maybe what I wrote was confusing. I meant you removed the `externalEndpoints` in this `Using Ceph storage backend` section. you might want to add the service override for this, like above for the `glanceAPIInternal`?

```
        override:
          service:
            metadata:
              annotations:
                metallb.universe.tf/address-pool: internalapi
                metallb.universe.tf/allow-shared-ip: internalapi
                metallb.universe.tf/loadBalancerIPs: 172.17.0.80
            spec:
              type: LoadBalancer
```

**Comment by fao89** [comment]
recheck

**Review by jistr — APPROVED** [review]

**Pull request merged**